net/tsdial/tsdial.go

```diff
@@ -129,6 +129,20 @@ func (d *Dialer) SetExitDNSDoH(doh string) {
 	}
 }
 
+// SetRoutes configures the dialer to dial the specified routes via Tailscale,
+// and the specified localRoutes using the default interface.
```

Comment:
are the localRoutes even needed? It's just localhost addresses and WSL2's weirdo local address. Neither will be coming from control.

Reply:
I think they're needed, as they also include LAN addresses if `ipn.Prefs.ExitNodeAllowLANAccess` is enabled:
https://github.com/tailscale/tailscale/blob/ce8969d82b5df9eff4f3ad71f6781c73e4481bf0/ipn/ipnlocal/local.go#L4196

Reply:
ah, okay